docs/development-guide.md

```diff
@@ -38,13 +40,7 @@ cd riff-docker
 make init-rtc
 make init-server
 ```
-for now (we need to set up `make init-signalmaster`)
-you also have to initialize the signalmaster working directory by:
-```
-cd ../signalmaster
-npm install
-cd ../riff-docker
-```
+
```

Comment:
yes but signalmaster still needs to be inited. see https://github.com/rifflearning/edu-docker/blob/master/docs/development-guide.md

basically add `make init-signalmaster` to the section above.

Reply:
Makes sense. I added ti to the end of the first command block.